getParams.m

```diff
@@ -24,8 +24,8 @@
 
 
 %% Debug mode settings
-cfg.debug               = false;  % To test the script
```

Comment:
i think we should keep it false and false here so by default, the opaque screen is used